### ISSUE TITLE
GeoBadges

### ISSUE THREAD

**Issue by geobrando** [maptime/maptime-admin#93]
It's a thing.  Is there a role for Maptime?

http://blog.mapstory.org/post/103049858841/announcing-geobadges-a-new-effort-to-inspire
http://www.geobadges.org/

**Comment by almccon** [comment]
Perhaps, but the word "verify" sounds hard to implement with Maptime. Are we just verifying attendance at a maptime meetup?

**Comment by geobrando** [comment]
Not sure.  Believe this is still in development.  @geomantic has had some involvment through [TeachOSM](http://teachosm.org/en/) and there's a [talk](http://stateofthemap.us/geobadges/) during SOTM.  Throwing it out there as I believe the important players should be in NYC and it's worth investigating.  But yeah, a credentialing system may be not be a good fit.

**Comment by geomantic** [comment]
@geobrando @almccon Yes, there is a role for Maptime. But it's not a Maptime session about GeoBadges. GeoBadges are credentials given to people who demonstrate a certain technical grasp of basic geographic workflow, say basic digitizing skills for a HOT activation. Since many young adults must complete service hours as a condition of graduating, geobadges are a convenient way to build a student resume and gain some applied geography skills by making certified, quality contributions to OpenStreetMap.    The interesting question is whether there are a series of Maptime courses that lead to a badge? Maptime could be a great vehicle for a series of staged courses that lead to a badge.

**Comment by geobrando** [comment]
While Maptime is by nature unstructured and chapters are encouraged to determine their own content, If I recall last year HQ promoted the idea of chapters following along a set "curriculum" (mainly a series of topics).  Can't recall exactly what all of the topics were, but this could be the basis for earning a credential.  If chapters chose to follow this series, it might also have the benefit of encouraging repeat attendance, which in my opinion is important in community building.

**Comment by almccon** [comment]
The suggested HQ curriculum is in this blog post: http://maptime.io/blog/2015/01/13/learning-together-in-2015/ 

The purple entries make up a five session curriculum that would repeat every six months or so.

**Comment by bethschechter** [comment]
I think it's totally a cool idea! The question is: how do we manage it? Or
do they manage it? Do you have thoughts on how this would be administered,
@geobrando?



//typed with thumbs

**Comment by geomantic** [comment]
@bethschechter @geobrando How we manage the geobadges is yet to be determined. As part of TeachOSM, we've had conversations w/ the NSF's GeoTech Center about hosting the credentials and alternatively, we're looking at a solution built on Credly.  Validation criteria and validation procedures are determined by the sponsoring organization (e.g. Maptime, OSM-US, HOT, MapGive, MissingMaps, etc.) and we're working out the mechanics of this. Jon Marino will be presenting on GeoBadges at SotMUS, and he should have a lot more details.

**Comment by geobrando** [comment]
Spoke a bit with @jonpmarino at State of the Map-US about geobadges.  Attended Liz Lyon's presentation and got a bit more information.  Unfortunately, I didn't take notes and for some reason video is not available.  Still unclear if a credentialing system is a good fit with maptime as it would entail a set curriculum.  I'll keep this issue open and report back as geobadges continues to develop.

**Comment by geobrando** [comment]
Article about an Open Badges implementation from CoderDojo

https://coderdojo.com/news/2015/04/16/why-open-badges-are-important-for-the-coderdojo-community/
